docs/CreateKoblenzHash.md

```diff
@@ -0,0 +1,69 @@
+# Creating Hashes for Koblenz Pass Data
+
+## Steps
+
+The example data is
+
+| Parameter    | Value         |
+|--------------|---------------|
+| Name         | Karla Koblenz |
+| Geburtstag   | 10.06.2003    |
+| Aktenzeichen | 123K          |
+
+
+### 1. Collect all data and merge it into an object
+```agsl
+//Example:
+full_name: "Karla Koblenz"
+extensions {
+  extension_region {
+    regionId: 95
+  }
+  extension_birthday {
+    birthday: 12213
+  }
+  extension_koblenz_pass_id {
+    pass_id: "123K"
+  }
+}
+```
```

Comment:
(Note that I might not have the full context)

I'd prefer if this new hash procedure here would not be based on CardInfo; instead just a new object format à la `{ fullName: "Karla Koblenz", birthday: 123123, passId: "123K" }` (also, `passId` might not make sense; it should be an identifier for the person and not the pass, I think?).
Otherwise, we cannot add other card specific fields (such as expiration dates, or similar) to the card info.

Reply:
Yes i agree it should  be sth like PersonInfoHash and not connected to Cardainfo

Reply:
Thanks for this hint, i was unsure how to handle this and change it now tot the new object format you suggested.